language/functional_tests/tests/testsuite/builtins/create_account_and_get_balance.mvir

```diff
@@ -1,22 +0,0 @@
-import 0x0.LibraAccount;
```

Comment:
This looks like a duplicate of the previous test, so deleting it makes sense.

Reply:
I copied the check for the balance in the other one